components/esp_modem/examples/modem_console/main/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 idf_component_register(SRCS "modem_console_main.cpp"
                             "console_helper.cpp"
+                            "my_module_dce.cpp"
```

Comment:
My module?

Reply:
rename to `custom_dce.cpp`

Reply:
Wanted to rename it and document, but realized that it's already documented and mentioned that this is a demonstration to create *my-module*. IMO this sounds better to me than "custom module", WYT?

see the docs here: [README.md](https://github.com/espressif/esp-protocols/blob/master/components/esp_modem/examples/modem_console/README.md)

Reply:
LGTM